connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerGroupMember.java

```diff
@@ -198,7 +199,7 @@ public void requestRejoin() {
     }
 
     public void maybeLeaveGroup(String leaveReason) {
-        coordinator.maybeLeaveGroup(leaveReason);
+        coordinator.maybeLeaveGroup(CloseOptions.GroupMembershipOperation.DEFAULT, leaveReason);
```

Comment:
I'm very unfamiliar with Connect but just based on the comment above where this method is called, it seems like this should probably be `LEAVE_GROUP` ?